packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -53,18 +53,22 @@ describe("Compile task", () => {
   const cacheDir = path.join("artifacts", "cache");
   const f1 = "asc-fee-check.copy.teal";
   const f2 = "asc-fee-check.teal";
+  const f3 = "gold-asa.py";
+  const f4 = "gold-asa.teal";
```

Comment:
Please add a `py` suffix to py files, eg: `f3-py`

Reply:
ok